acme_diags/driver/utils/dataset.py

```diff
@@ -278,6 +278,10 @@ def _find_climo_file(self, path_name, data_name, season):
         """
         Locate climatology file name based on data_name and season.
         """
+        if os.path.isfile(path_name):
+            # Without this line, lat_lon_vector plots fail to show seasonal plots.
+            # TODO: However, the seasonal plots look identical to the ANN plots. Why?
```

Comment:
The seasonal plots look identical to the annual plots. Is something set up wrong?

Reply:
I think the following lines in `run.py` caused the problem:

```
param.test_data_name = '20161118.beta0.F1850COSP.ne30_ne30.edison_ANN_climo.nc'
param.ref_data_name = '20161118.beta0.F1850COSP.ne30_ne30.edison_ANN_climo.nc'
param.test_name = '20161118.beta0.F1850COSP.ne30_ne30.edison_ANN_climo.nc'
param.ref_name = '20161118.beta0.F1850COSP.ne30_ne30.edison_ANN_climo.nc' 
```
This way the input data were hard-coded to use ANN for all. Replace those with below two lines, should work...
```
param.test_name = '20161118.beta0.F1850COSP.ne30_ne30.edison`
param.ref_name = '20161118.beta0.F1850COSP.ne30_ne30.edison`